src/crate/client/test_connection.py

```diff
@@ -23,7 +25,25 @@ def test_invalid_server_version(self):
         self.assertEqual((0, 0, 0), connection.lowest_server_version.version)
         connection.close()
 
-    def test_with_is_supported(self):
+    def test_context_manager(self):
```

Comment:
`context_manager`? isn't `test_connection` more appropriate?

Reply:
Dear Marios,

the _whole module_ is about testing the `connect()` function in different variants. This specific case exercises the use of the `connect()` method as a context manager.

With kind regards,
Andreas.

Reply:
Thx for clarifying!